docs/content.go

```diff
@@ -565,20 +571,36 @@ Enterprise Performance Computing (EPC)`
   $ singularity pull tensorflow.sif docker://tensorflow/tensorflow:latest
 
   From Shub
-  $ singularity pull singularity-images.sif shub://vsoch/singularity-images`
+  $ singularity pull singularity-images.sif shub://vsoch/singularity-images
+
+  From supporting OCI registry (e.g. Azure Container Registry)
+  $ singularity pull image.sif oras://<username>.azurecr.io/namespace/repo:tag`
```

Comment:
#1. Can we put a real example here?
#2. `repo` vs `image` again.

Reply:
1. If we want a public acr registry for people to pull from to try out, then we would need to set that up ourselves. I tested using a trial and am not planning on making the registry public or extending it once it expires.

2. See above.